Configuration/StandardSequences/python/L1TrackTrigger_cff.py

```diff
@@ -4,7 +4,7 @@
 from SimTracker.TrackTriggerAssociation.TrackTriggerAssociator_cff import *
 from L1Trigger.TrackerDTC.ProducerED_cff import *
 
-#L1TrackTrigger=cms.Sequence(TrackTriggerClustersStubs*TrackTriggerAssociatorClustersStubs*TrackTriggerTTTracks*TrackTriggerAssociatorTracks)
-L1TrackTrigger=cms.Sequence(TrackTriggerClustersStubs*TrackTriggerAssociatorClustersStubs*TrackerDTCProducer)
+L1TrackTrigger=cms.Sequence(TrackTriggerClustersStubs*TrackTriggerAssociatorClustersStubs*TrackerDTCProducer*TrackTriggerTTTracks*TrackTriggerAssociatorTracks)
```

Comment:
I don't see any definition of `TrackTriggerTTTracks` in CMSSW

Reply:
sorry for jumping in, but may I ask the reason of the `TrackTriggerAssociatorTracks` in the main L1TrackTrigger sequence ?
it is calling https://github.com/cms-sw/cmssw/blob/6b9ccb57dee572818384626c2db944f300b1fd15/SimTracker/TrackTriggerAssociation/python/TTTrackAssociation_cfi.py

thanks

Reply:
right....
we got to use these  https://github.com/cms-sw/cmssw/blob/master/L1Trigger/TrackFindingTracklet/python/Tracklet_cfi.py#L3

Reply:
my previous comment was to @silviodonato

Reply:
@mtosi 
This is what we have ran in the configuration given to us by TrackTrigger community, 
https://github.com/cms-l1t-offline/cmssw/blob/phase2-l1t-integration-CMSSW_11_1_0_pre6/L1Trigger/L1TCommon/test/reprocess_L1TrackTrigger_L1T.py#L178

If someone from the TrackTrigger community would prefer to make a PR putting the TrackTrigger in the sequence in a preferred way, that would be helpful.  (attn @skinnari )

Reply:
The commit 2e633c1 should properly addresses the questions discussed above.